iocore/cache/CacheTest.cc

```diff
@@ -556,32 +556,32 @@ test_RamCache(RegressionTest *t, RamCache *cache, const char *name, int64_t cach
   for (int l = 0; l < 10; l++) {
     for (int i = 0; i < 200; i++) {
       IOBufferData *d = THREAD_ALLOC(ioDataAllocator, this_thread());
-      INK_MD5 md5;
+      CryptoHash hash;
 
       d->alloc(BUFFER_SIZE_INDEX_16K);
       data.push_back(make_ptr(d));
-      md5.u64[0] = ((uint64_t)i << 32) + i;
-      md5.u64[1] = ((uint64_t)i << 32) + i;
-      cache->put(&md5, data[i].get(), 1 << 15);
+      hash.u64[0] = ((uint64_t)i << 32) + i;
+      hash.u64[1] = ((uint64_t)i << 32) + i;
```

Comment:
This seems to have the built in assumption of a 128 bit hash value.

Reply:
Yes, but 1) this is a test and this is a synthetic key 2) it is still OK since the second 128 are initialized to 0 and that is still a valid key and the only bits which are actually used are the first 128.